### PR TITLE
Wrench cleanup and perspective support

### PR DESCRIPTION
**Pull request by kvark**
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/839)

### CODE REVIEW

File: wrench/src/yaml_frame_reader.rs [hunk 1]
```diff
@@ -492,6 +493,10 @@ impl YamlFrameReader {
         let mix_blend_mode = yaml["mix-blend-mode"].as_mix_blend_mode().unwrap_or(MixBlendMode::Normal);
         let sc_full_rect = LayoutRect::new(LayoutPoint::new(0.0, 0.0), bounds.size);
         let clip = self.to_clip_region(&yaml["clip"], &sc_full_rect, wrench).unwrap_or(ClipRegion::simple(&sc_full_rect));
+        //let clip = ClipRegion::simple(&sc_full_rect);
```
Comment (glennw) [inline wrench/src/yaml_frame_reader.rs]: Can we remove these comments?

File: wrench/src/yaml_helper.rs [hunk 2]
```diff
@@ -237,12 +237,22 @@ impl YamlHelper for Yaml {
                                                                             1.0,
                                                                             Radians::new(theta));
 
-                    return Some(pre_transform.pre_mul(&transform).pre_mul(&post_transform))
+                    Some(pre_transform.pre_mul(&transform).pre_mul(&post_transform))
+                }
+                ("project", ref args) if args.len() == 1 => {
```
Comment (glennw) [inline wrench/src/yaml_helper.rs]: I think this should be ```"perspective"``` which is what CSS uses.

Comment (kvark) [inline wrench/src/yaml_helper.rs]: > I think this should be "perspective" which is what CSS uses.

Note that yaml would look like `perspective: perspective(100)`

**Comment by kvark** [comment]
Fixes applied, waiting for CI.

**Comment by glennw** [comment]
Looks good, thanks!

**Pull request merged**